changelog.json

```diff
@@ -1,5 +1,19 @@
 {
   "releases": {
+    "2.4.1": [
+      "[Added] Indicate which commits are un-pushed in \"History\" tab - #5873",
```

Comment:
```suggestion
      "[Added] Indicate which commits have not yet been pushed to the remote in \"History\" tab - #5873",
```

Reply:
I also personally think this is more of an ”Improved” but it might be the main feature of this update

Reply:
Thanks @Rexogamer. The primary reason for the release is to address the Git security vulnerability. I agree this one could go either way, but we're fine with "Added". Thanks.